### PR TITLE
Updated: remove _blank

### PR DESCRIPTION
**Pull request by mfkitty**

### CODE REVIEW

File: layout/includes/sidebar.pug [hunk 1]
```diff
@@ -25,24 +25,24 @@
     .author-info__description.text-center= config.description
     if theme.follow && theme.follow.enable
       .follow-button
-        a(href=theme.follow.url target="_blank")= theme.follow.text || 'Follow Me'
+        a(href=theme.follow.url)= theme.follow.text || 'Follow Me'
```
Comment (Molunerfinn) [inline layout/includes/sidebar.pug]: `Follow` often jump to other site. So I think keeping `blank` is better.

File: layout/includes/sidebar.pug [hunk 2]
```diff
@@ -25,24 +25,24 @@
...
-          span.pull-left= _p('sidebar.categories') 
+          span.pull-left= _p('sidebar.categories')
           span.pull-right= site.categories.length
     if theme.links_title
       hr
       .author-info-links
         .author-info-links__title.text-center= theme.links_title
         each url, name in theme.links
-          a.author-info-links__name.text-center(href=url target="_blank")= name
+          a.author-info-links__name.text-center(href=url)= name
```
Comment (Molunerfinn) [inline layout/includes/sidebar.pug]: `Link` often jump to other site. So I think keeping blank is better.

File: layout/includes/nav.pug [hunk 3]
```diff
@@ -9,7 +9,7 @@ nav#nav(style=bg_img class=flag)
     if(theme.social)
       #site-social-icons
         each url, icon in theme.social
-          a.social-icon(href=url target="_blank")
+          a.social-icon(href=url)
```
Comment (Molunerfinn) [inline layout/includes/nav.pug]: `Social icon` often jump to other site. So I think keeping blank is better.

**Comment by mfkitty** [comment]
@Molunerfinn 

Do we really need to force users to stay on our site? When we use google search, it will list the results, we can click one and go to other site, just like going to your Blog from [Github profile](https://github.com/Molunerfinn). If users want to stay here, they can still press the `Ctrl`. I think the point is that they can choose.

On the other hand, if we keep `_blank` on navigation, follow and links, and `_self` for others, the behavior is not unified. But these places are not forms, don’t need to keep users' input, so why we to do this?

If you insist on this, I can update this PR.

**Comment by Molunerfinn** [comment]
@mfkitty Hmmm, you're right. My thought is a bad reason. Thank you for your contribution!

**Pull request merged**